### ISSUE TITLE
NTR: negative regulation of autoproteolysis

### ISSUE THREAD

**Issue by sfmcguinness** [geneontology/go-ontology#14341]
- Potential GO term name: negative regulation of autoproteolysis
- Definition: Any process that stops, prevents or reduces the frequency, rate or extent of the hydrolysis of a peptide bond or bonds within a protein catalyzed by the protein itself.
- Aspect: Biological Process
-Relationships: 
(Parent Term) GO:0045861 negative regulation of proteolysis
-References: PMID 23303392
-Synonyms: negative regulation of self-proteolysis; negative regulation of autocleavage

**Comment by jimhu-tamu** [comment]
This is for CACAO.

**Comment by vanaukenk** [comment]
@sfmcguinness and @jimhu-tamu 

I'm looking at this part of the ontology, and seeing two different possible parent terms, neither of which are 'autoproteolysis':

GO:0019538 protein metabolic process
-GO:0006508 proteolysis
--GO:0097264 self proteolysis
---GO:1990092 calcium-dependent self proteolysis
---GO:1990091 sodium-dependent self proteolysis

GO:0019538 protein metabolic process
-GO:0051604 protein maturation
-GO:0006508 proteolysis
--GO:0016485 protein processing
--GO:0016540 protein autoprocessing

The definitions are:

GO:0097264 self-proteolysis
The hydrolysis of proteins into smaller polypeptides and/or amino acids by cleavage of their own peptide bonds. Source: GOC:yaf, PMID:18676612, PMID:19144634

GO:0016540 protein autoprocessing
Processing which a protein carries out itself. This involves actions such as the autolytic removal of residues to generate the mature form of the protein. Source: GOC:ai, PMID:9335337

@ukemi - would it make sense to move GO:0097264 to be an is_a child of GO:0016540?

Also, @jimhu-tamu  - Do the CACAO students use annotation extensions?  I am wondering here about the correct usage and interpretation of the proposed term 'negative regulation of autoproteolysis'.

This paper sounds like a good candidate for a GO-CAM model.

**Comment by ukemi** [comment]
> would it make sense to move GO:0097264 to be an is_a child of GO:0016540?

It would only make sense if self-proteolysis always leads to the mature form of a protein. In any other context, we would break the true path rule.

**Comment by ukemi** [comment]
PMID:24779472 suggests that it would break the true path rule.

**Comment by vanaukenk** [comment]
Okay, thanks @ukemi 

@jimhu-tamu and @sfmcguinness since we don't have 'autoproteolysis'  but instead have 'self proteolysis' here's my proposal for term name and placement in the ontology:

GO:0045861 negative regulation of proteolysis
-GO:new negative regulation of self proteolysis

Synonyms: negative regulation of self-proteolysis; negative regulation of autocleavage, negative regulation of autoproteolysis, negative regulation of autolysis

I would also propose to add synonyms to self proteolysis (GO:0097264):

Existing synonyms: self-proteolysis, autolysis
New synonyms: autocleavage, autoproteolysis

Does that look okay to you?

**Comment by jimhu-tamu** [comment]
yes but I would change the synonyms to remove autolysis.  Autolysis is not a synonym for autoproteolysis, since autolysis is usually refers to a process at the cellular level.

https://en.wikipedia.org/wiki/Autolysis_(biology)

**Comment by vanaukenk** [comment]
Thanks, @jimhu-tamu @sfmcguinness 

In looking at the parentage for the new term, I see that it would map up to the 'cellular protein metabolic process' branch of the BP, but wouldn't have any relation to the 'viral processes' branch of the BP.  Should the new term instead be under the 'viral processes' branch?

<img width="857" alt="screen shot 2017-10-25 at 3 21 02 pm" src="https://user-images.githubusercontent.com/1730534/32018753-dbc0236e-b998-11e7-9c18-f4cc27c05c78.png">

If so, then perhaps the new term should instead be a child of something like GO:0046726 positive regulation by virus of viral protein levels in host cell:

<img width="719" alt="screen shot 2017-10-25 at 3 30 28 pm" src="https://user-images.githubusercontent.com/1730534/32019139-3c9abc3e-b99a-11e7-920b-63ce36c42224.png">

Although there are only four experimentally based annotations to GO:0046726 and they are all to human proteins, so I'm a bit uncertain about how this term was intended to be used.

@pgaudet  - would annotating a viral protein to a term like GO:0046726 (or a more specific child term like 'negative regulation by virus of viral protein self proteolysis') be consistent with how the viral group would interpret its meaning?

In summary, I think the biology here is:

Hex (phage protein) -| RecA (bacterial protein) -> 434 (phage protein) self proteolysis

@jimhu-tamu @sfmcguinness - is that right?

Sorry this has gotten complicated, but I just want to make sure we get the biology and ontology parentage correct.

**Comment by jimhu-tamu** [comment]
It always gets complicated! Your diagram looks right to me. But note that the 
- RecA stimulation of autocleavage is itself regulated by RecA activities/SOS response to DNA damage
- The RecA stimulated autocleavage happens for both viral and host proteins. RecA stimulates cleavage of LexA, the repressor of the SOS response, and several other proteins, including subunit of an error-prone translesion DNA polymerase.

**Comment by pgaudet** [comment]
Hi @vanaukenk 

Most annotations to the terms: 

regulation by virus of viral protein levels in host cell
positive regulation by virus of viral protein levels in host cell
negative regulation by virus of viral protein levels in host cell
are indeed to host proteins and therefore wrong; I will challenge them.

**Comment by jimhu-tamu** [comment]
Hi @pgaudet 

What annotations to host proteins are you referring to? Not the original annotation we want to make, I hope.

**Comment by pgaudet** [comment]
Hi @jimhu-tamu 

Most annotations are to host proteins: 
TAIR:locus:2184362	AT5G04430	GO:0046719	TAIR	NCBITaxon:3702	IMP	TAIR:Publication:501727403|PMID:18762309	20080929
UniProtKB:Q96BZ9	TBC1 domain family member 20	GO:0046726	AgBase	NCBITaxon:9606	IMP	PMID:17686842	20141112
UniProtKB:Q16531	DNA damage-binding protein 1	GO:0046726	AgBase	NCBITaxon:9606	IMP	PMID:23137809	20141218
UniProtKB:P42224	Signal transducer and activator of transcription 1-alpha/beta	GO:0046725	AgBase	NCBITaxon:9606	IMP	PMID:15825084	20141112
UniProtKB:P63172	Dynein light chain Tctex-type 1	GO:0019060	UniProt	NCBITaxon:9606	IMP	PMID:18647839	20100507
UniProtKB:P09914	Interferon-induced protein with tetratricopeptide repeats 1	GO:0019060	BHF-UCL	NCBITaxon:9606	IDA	PMID:19008854	20101230
UniProtKB:Q9P035	Very-long-chain (3R)-3-hydroxyacyl-CoA dehydratase 3	GO:0046726	AgBase	NCBITaxon:9606	IMP	PMID:18160438	20150115
UniProtKB:P67868	Casein kinase II subunit beta	GO:0046719	AgBase	NCBITaxon:9913	IMP	PMID:19692545	20160126
UniProtKB:O95793	Double-stranded RNA-binding protein Staufen homolog 1	GO:0046726	AgBase	NCBITaxon:9606	IMP	PMID:23907398	20141112

**Comment by jimhu-tamu** [comment]
@pgaudet 

Initially that makes sense, and a quick scan suggests to me that many are wrong. But it may hard to tell just by looking at the gaf for the following reason: 

if we have a process that is a chain of regulatory steps, wouldn't we annotate both host and viral proteins involved? 

In the paper being curated by @sfmcguinness, hex is a viral protein, which regulates RecA, a host protein, which regulates cI, a viral protein, which regulates a program of viral gene expression. From a process perspective, shouldn't RecA be part of regulation by virus of protein levels in host cell? (Or whatever we use for regulation of viral transcription).

If yes, then some or all of those might be correct.

**Comment by pgaudet** [comment]
Hi @jimhu-tamu 

I don't know why we came on the topic of viral process for this term in fact - why was this suggested already ?

Thanks, Pascale

**Comment by jimhu-tamu** [comment]
I'm confused too.

**Comment by pgaudet** [comment]
;) @vanaukenk ? Help please !

**Comment by vanaukenk** [comment]
Hi @pgaudet and @jimhu-tamu 

The reason the viral processes issue came up is that with the first suggested placement of the requested term in the ontology, there would have been no indication, by following the parentage, that what was being captured here was a viral process (if, indeed it is - I thought it was).

See my comment above with the screenshots of the ontology in AmiGO.

**Comment by pgaudet** [comment]
Thanks @vanaukenk ! 

So looking at the original reference (PMID:23303392): let me see if I understand the biology: 
* **Bacterial RecA** normally triggers phage proteins to perform autoproteolysis, so this is some sort of "defense response to virus", and it should also be OK to annotate to "positive regulation of autoproteolysis" (which in this case is *not* a viral process). You could even use the term '**negative regulation by host of viral transcription**' which is the phage target of RecA. 

* **Phage Hex** negatively regulates RecA. This could captured under 'modulation by symbiont of host defense response'; perhaps under that term you want to create 'inhibition by virus of negative regulation by host of viral transcription' (but that's a mouthful! and a Noctua model probably). 
Another way to capture the function of phage Hex could be under 'suppression by virus of host molecular function', under which we could have something like '**suppression by virus of host protease activator activity**'. 

If that makes sense, then the only term you need to create would be 'suppression by virus of host protease activator activity'.

Thanks, Pascale

**Comment by vanaukenk** [comment]
I like this suggestion:

Another way to capture the function of phage Hex could be under 'suppression by virus of host molecular function', under which we could have something like 'suppression by virus of host protease activator activity'.

@jimhu-tamu @sfmcguinness  What do you think?
